aten/src/ATen/native/native_functions.yaml

```diff
@@ -9084,17 +9084,17 @@
 
 - func: _histogramdd_bin_edges(Tensor self, int[] bins, *, float[]? range=None, Tensor? weight=None, bool density=False) -> Tensor[]
   dispatch:
-    CPU: histogramdd_bin_edges_cpu
+    CPU, Meta: histogramdd_bin_edges_cpu
```

Comment:
Similar to `eye`, which shares Meta and CPU. `histogramdd` has a bunch of checks, so I didn't want to split these.